editions/tw5.com/tiddlers/saving/Saving with TiddlyFox.tid

```diff
@@ -5,22 +5,22 @@ created: 20131221085742684
 delivery: Browser Extension
 description: Browser extension for older versions of Firefox
 method: save
-modified: 20200507105421421
+modified: 20230803051244464
 tags: Saving Firefox
 title: Saving with TiddlyFox
 type: text/vnd.tiddlywiki
 
```

Comment:
Imo this tiddler should only contain: `<<.deprecated-since "FireFox 57" "Saving with FireFox">>` 

and with a new PR it should link to a new tiddler "Saving with FireFox", which needs to be written, using the elements which still work as described here.

Once "Saving with FireFox" the existing tiddler "GettingStarted - Firefox" should transclude it. It transcludes "this tiddler" at the moment.

Reply:
Done and done.